src/gps/GPS.cpp

```diff
@@ -64,16 +64,23 @@ bool GPS::setupGPS()
     if (_serial_gps && !didSerialInit) {
         didSerialInit = true;
 
+#if CONFIG_IDF_TARGET_ESP32S3
+    // In esp32s3 framework, setRxBufferSize needs to be initialized before Serial
+    _serial_gps->setRxBufferSize(2048); // the default is 256
+#endif
+
 // ESP32 has a special set of parameters vs other arduino ports
 #if defined(GPS_RX_PIN) && defined(ARCH_ESP32)
         _serial_gps->begin(GPS_BAUDRATE, SERIAL_8N1, GPS_RX_PIN, GPS_TX_PIN);
 #else
         _serial_gps->begin(GPS_BAUDRATE);
 #endif
-#ifdef ARCH_ESP32
+
+#if CONFIG_IDF_TARGET_ESP32
         _serial_gps->setRxBufferSize(2048); // the default is 256
 #endif
-#ifdef TTGO_T_ECHO
+
+#if defined(TTGO_T_ECHO) || defined(LILYGO_TBEAM_S3_CORE) 
```

Comment:
Not sure this is neccessary for the new board. We've had a few firmware versions in the past where on the T-Echo only (!) baud of the onboard gps chip was reduced to 4800. This reverts that persistent change, and i'm rather sure this does not apply to the new t-beam, except if the new board uses the L76K chip as well. The 4800-9600 dance can be skipped regardless.

Reply:
It can be used like that, but to avoid misunderstanding, I still fix it to 9600